core/src/main/java/tc/oc/pgm/action/actions/TeamAliasAction.java

```diff
@@ -0,0 +1,33 @@
+package tc.oc.pgm.action.actions;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.party.Party;
+import tc.oc.pgm.teams.Team;
+
+public class TeamAliasAction extends AbstractAction<Match> {
+  private final String teamId;
```

Comment:
Should not be a string, this should be a reference to a team, will leave more details at parsing